dev/com.ibm.ws.microprofile.rest.client_fat/fat/src/com/ibm/ws/microprofile/rest/client/fat/AsyncMethodTest.java

```diff
@@ -68,7 +72,65 @@ public static void afterClass() throws Exception {
             assertTrue("Found JsonBProvider errors in log file", 
                        jsonbProviderErrors == null || jsonbProviderErrors.isEmpty());
         } finally {
+            server.dumpServer("dump.zip");
             server.stopServer("CWWKF0033E"); //ignore this error for mismatch with jsonb-1.0 and Java EE 7
         }
     }
+
+    @Before
+    public synchronized void obtainServerDumps() throws Exception {
+        if (server.isStarted()) {
+            new Thread(()-> {
+                synchronized(AsyncMethodTest.this) {
+                    if (testRunning && server.isStarted()) {
+                        try {
+                            server.dumpServer("dump1");
+                        } catch (Exception e) {
+                            e.printStackTrace();
+                        }
+                    } else {
+                        return;
+                    }
+                }
+                try {
+                    Thread.sleep(1000);
```

Comment:
might be helpful to parameterize the sleep interval between dumps

Reply:
Opened issue #6946 to address this more broadly.  Thanks for the suggestion, Gary!